Formula/discount.rb

```diff
@@ -38,8 +38,13 @@ def install
       --enable-dl-tag
       --enable-pandoc-header
       --enable-superscript
+      --pkg-config
+      --shared
     ]
     system "./configure.sh", *args
+    # Homebrew-specific workaround to stop build script from trying to
+    # call ldconfig, which requires root access to work.
+    inreplace "librarian.sh", "/sbin/ldconfig", "true" if OS.linux?
```

Comment:
We should not modify the scripts. If it is an upstream bug, they can fix it. If it is the behavior upstream thinks desirable, we should keep it.